InferenceSystem/infra/Write-Log.ps1

```diff
@@ -0,0 +1,19 @@
+function Write-Log {
+    [CmdletBinding()]
+    param(
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string]$Message,
+ 
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [ValidateSet('Information','Warning','Error')]
+        [string]$Severity = 'Information'
+    )
+ 
+    [pscustomobject]@{
+        Time = (Get-Date -f g)
+        Message = $Message
+        Severity = $Severity
+    } | Export-Csv -Path "$env:Temp\LogFile.csv" -Append -NoTypeInformation
+ }
```

Comment:
No newline at end of file though I don't know if it's good or bad practice for PS scripts.